Project/Project/CDTViewController.m

```diff
@@ -25,7 +25,7 @@ @interface CDTViewController ()
 
 @property (readonly) CDTDatastore *datastore;
 @property (readonly) CDTTodoReplicator *todoReplicator;
-@property (nonatomic,strong) NSArray *taskRevisions;
+@property (nonatomic, strong) NSArray *taskRevisions;
```

Comment:
This variable should be renamed to reflect the fact it is now an array of `CDTTodo`s